relation_engine_server/test/test_spec_loader.py

```diff
@@ -24,7 +24,8 @@ def test_get_names(self, schema_type_names=[], expected=[]):
 
         # this method should only be run from another test method
         if len(schema_type_names) == 0:
-            self.skipTest('No schema type names supplied. Skipping')
+            self.assertTrue(True)
+            return
```

Comment:
Same effect as `skipTest` but without giving a misleading `s` in the test output